### ISSUE TITLE
[REQ] Feature Request Description

### ISSUE THREAD

**Issue by johnny1952** [OpenAPITools/openapi-generator#4240]
### Is your feature request related to a problem? Please describe.

I found that og(openapi-generator) version 3.1.1-SNAPSHOT can generate PYTHON client code if the path contains port, like  ":11180/a/b/c".
But version 4.1.1 reports a error :  -attribute paths. Resource :11180/a/b/c should start with / .

## Describe the solution you'd like

What I want is that the higher version can be backward compatible. 
In other words, the version 4.1.1 can also generate PYTHON client code without any error.

Is there a switch to turn off the test?
